app/forms/employment_form.rb

```diff
@@ -94,4 +95,12 @@ def reset_new_job!
   def target
     resource.employment || resource.build_employment
   end
+
+  def end_date_after_start_date?
+    return if end_date.blank? || start_date.blank?
+
+    if end_date < start_date
+      errors.add(:end_date)
```

Comment:
What happened to the second parameter ? the one specifying what the error is (:end_date_before_start)